lib/render_csv/csv_renderable.rb

```diff
@@ -17,7 +18,7 @@ def to_csv(options = {})
       columns += options[:add_methods].map(&:to_s) if options[:add_methods]
 
       CSV.generate(encoding: 'utf-8') do |row|
-        row << columns
+        row << options[:header] ||= columns
```

Comment:
I don't think you need `||=` since that's going to mutate the options hash. What about doing `options.fetch(:header, columns)`?